extensions/2.0/Vendor/EXT_lights_image_based/schema/scene.EXT_lights_image_based.schema.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_lights_imageBased scene extension",
+    "type": "object",
+    "properties": {
+        "light": {
+            "allOf": [
+                {
+                    "$ref": "glTFid.schema.json"
+                }
+            ],
+            "description": "The id of the light referenced by this scene."
+        }
+    },
```

Comment:
`light` should be a required field to disallow empty extension object
```json
{
  "asset": {
    "version": "2.0",
    "extesnionsUsed": ["EXT_lights_image_based"]
  },
  "scenes": [
    {
      "extesnions": {
        "EXT_lights_image_based": {}
      }
    }
  ]
}
```

Reply:
Done